### DISCUSSION TITLE
Difference between Helics and OpenDSS

### DISCUSSION THREAD

**Discussion by christopinka** [GMLC-TDC/HELICS#2525] [General]
Is Helics a replacement for OpenDSS? I see it mentioned here - https://docs.helics.org/en/latest/references/Tools_using_HELICS.html

Can Helics on it's own be used for Distribution System Simulation? Are there examples of Helics use for this? 

Thanks

Chris

**Comment by trevorhardy** [comment]
HELICS and OpenDSS have different purposes and are not interchangeable. OpenDSS is a distribution power system simulator and HELICS is a co-simulation tool. You could use HELICS in conjunction with OpenDSS to allow OpenDSS to send and receive information with, say, a transmission system simulator, building a larger model of the power system.
